colibre/registration.py

```diff
@@ -175,6 +215,24 @@
         ),
     )
 
+# Get HI to dust ratio
+try:
+    dust_to_hi_ratio = total_dust_mass / HI_mass
+    dust_to_hi_ratio.name = "M_{{\\rm dust}}/M_{{\\rm HI}}"    
+    setattr(self, "gas_dust_to_hi_ratio", dust_to_hi_ratio)
```

Comment:
Aperture information? If not in an aperture, but uses FOF halo, would be nice to have a comment here explicitly stating that.

Reply:
added comment as above